lib/augmentConfig.js

```diff
@@ -178,7 +178,13 @@ async function extendConfig(stylelint, config, configDir, rootConfigDir, filePat
 	let resultConfig = originalWithoutExtends;
 
 	for (const extendLookup of normalizedExtends) {
-		const extendResult = await loadExtendedConfig(stylelint, configDir, extendLookup);
+		let extendResult;
+
+		if (typeof extendLookup === 'string') {
```

Comment:
minor

```js
if (typeof extendLookup === 'object' && extendLookup !== null) {
// …
} else {
// …
}
```
should be enough.
i.e. if we really wanted to check the type we would use something like [is-string](https://www.npmjs.com/package/is-string) or [isstring](https://www.npmjs.com/package/@string/isstring)

Reply:
Strictly checking the type here is important to avoid unexpected failures. Also, it's too much to install another package just for `typeof`.

Reply:
related: #5341